cd/before-deploy.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+if [ "$TRAVIS_BRANCH" = 'master' ] && [ "$TRAVIS_PULL_REQUEST" == 'false' ]; then
```

Comment:
We only want to setup and execute the deployment on builds on the master branch.  In travis, when you open a release PR into master, it spawns 3 builds: one on the branch, one on the merged branch before the merge, and one on the `master` branch after the merge.  This just picks out only the last, post-merge build.